notebooks/week3/01.feature_serving.py

```
@@ -0,0 +1,333 @@
+spark.sql(f"""
+    ALTER TABLE {feature_table_name}
+    SET TBLPROPERTIES (delta.enableChangeDataFeed = true)
+""")
+
+# COMMAND ----------
+
+online_table_name
+
+# COMMAND ----------
+
+from pprint import pprint
```

Ruff (0.8.2)

Module level import not at top of file

(E402)